Components/Linux/wordpress-recipes-linux/install_wordpress_with_db.yml

```diff
@@ -0,0 +1,98 @@
+name: InstallWordPressWithDb
+description: this document installs Wordpress with Db on Amazon Linux
```

Comment:
Similar to above, `This document installs WordPress with a database on Amazon Linux 2`